tests/test_mrxs.py

```diff
@@ -0,0 +1,115 @@
+from PIL import Image
+import os
+import urllib.request
+from fastapi import APIRouter
+from pims.formats import FORMATS
+import io
+from pims.importer.importer import FileImporter
+
+from tests.utils.formats import info_test, thumb_test, resized_test, mask_test, crop_test, crop_null_annot_test, histogram_perimage_test
+
+from pims.files.file import (
+    EXTRACTED_DIR, HISTOGRAM_STEM, ORIGINAL_STEM, PROCESSED_DIR, Path,
+    SPATIAL_STEM, UPLOAD_DIR_PREFIX, unique_name_generator
+)
+from pims.files.archive import Archive, ArchiveError
+
+import pytest
+
+def get_image(path, image):
```

Comment:
Is is possible to use the same naming convention as in the tests in the main `pims` repo ? (e.g `filename` instead of `image`)